### PR TITLE
feat: Bump cloudposse-s3-log-storage version

### PR DESCRIPTION
**Pull request by Balsir**
# Description

Bump version to enable first-try bucket creation. Recommended configuration requires ACL (therefore, ACL creation shouldn't be disabled.

## Type of change

- [ ] A bug fix (PR prefix `fix`)
- [x] A new feature (PR prefix `feat`)
- [ ] A code change that neither fixes a bug nor adds a feature (PR prefix `refactor`)
- [ ] Adding missing tests or correcting existing tests (PR prefix `test`)
- [ ] Changes that do not affect the meaning of the code like white-spaces, formatting, missing semi-colons, etc. (PR prefix `style`)
- [ ] Changes to our CI configuration files and scripts (PR prefix `ci`)
- [ ] Documentation only changes (PR prefix `docs`)

### CODE REVIEW

**Review by dojci — CHANGES_REQUESTED** [review]
Please fix the failing pipeline or renovate actions

**Comment by Balsir** [comment]
Tested in tf-org, plan adds expected resource, because versioning is disabled:
````shell
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the
following symbols:
  + create

Terraform will perform the following actions:
  # module.ssm_session_log.module.ssm-bucket.module.aws_s3_bucket.aws_s3_bucket_versioning.default[0] will be created
  + resource "aws_s3_bucket_versioning" "default" {
      + bucket = "ref-management-ssm"
      + id     = (known after apply)

      + versioning_configuration {
          + mfa_delete = (known after apply)
          + status     = "Suspended"
        }
    }

Plan: 1 to add, 0 to change, 0 to destroy.
````

File: variables.tf [hunk 1]
```diff
@@ -133,19 +94,15 @@ variable "kms_alias" {
...
-  default     = false
-}
-
 variable "cloudwatch_group_enabled" {
   description = "Set to `false` to prevent the module from creating CloudWatch log group"
   type        = bool
   default     = false
 }
 
+# tflint-ignore: terraform_unused_declarations
```
Comment (dojci) [inline variables.tf]: Please remove all unused values

**Review by dojci — CHANGES_REQUESTED** [review]

**Review by dojci — APPROVED** [review]
LGTM

**Pull request merged**